### ISSUE TITLE
gist_password_salt Bug Report

### ISSUE THREAD

**Issue by meetkool** [meetkool/dkforest#5]
The code you provided does not have any bugs, as it correctly defines a function to capitalize the first letter of a word and then tests this function with an example input. However, the function's name is somewhat misleading, as `title()` not only capitalizes the first letter of the word but also capitalizes the first letter of each word in a string. If you want to capitalize only the first letter of the word, you can use the `capitalize()` method instead. Here's the corrected code:

```python
def capitalize_first(word):
    return word.capitalize()


print(capitalize_first("hello world"))  # Output: "Hello world"
```

In summary, the original code works as intended, but the function's name is misleading. The corrected code uses the appropriate method and has a consistent output format.
